test/parallel/sdk.API.getInfo.test.ts

```
@@ -0,0 +1,190 @@
+  expect(secondAssetId).bnEqual(token1);
+});
+
+test("check parameters of getLiquidityTokenIds function", async () => {
+  const tokenIds: BN[] = [];
+
+  const tokenIdsString = await mangata.query.getLiquidityTokenIds();
+  const liqIdString = liqId.toString();
+
+  for (let index = 0; index < tokenIdsString.length; index++) {
+    if (tokenIdsString[index] === liqIdString) {
+      tokenIds.push(stringToBN(tokenIdsString[index]));
+    }
+  }
+
+  expect(tokenIds[0]).bnEqual(liqId);
+});
+
+test("check parameters of getPool function", async () => {
+  const liqPool = await mangata.query.getPool(liqId.toString());
+
+  expect(liqPool.firstTokenId).toEqual(MGA_ASSET_ID.toString());
+  expect(liqPool.secondTokenId).toEqual(token1.toString());
+  expect(liqPool.liquidityTokenId).toEqual(liqId.toString());
+});
+
+test("check parameters of getPools function", async () => {
+  const liqPools = await mangata.query.getPools();
+  const liqPoolsFiltered: PoolWithRatio[] = [];
+
+  for (let index = 0; index < liqPools.length; index++) {
+    if (liqPools[index].liquidityTokenId === liqId.toString()) {
+      liqPoolsFiltered.push(liqPools[index]);
+    }
+  }
+
+  expect(liqPoolsFiltered[0].firstTokenId).toEqual(MGA_ASSET_ID.toString());
+  expect(liqPoolsFiltered[0].secondTokenId).toEqual(token1.toString());
+  expect(liqPoolsFiltered[0].liquidityTokenId).toEqual(liqId.toString());
+});
+
+test("check parameters of getTotalIssuance functions", async () => {
+  const valueIssuance = await mangata.query.getTotalIssuance(liqId.toString());
+  const valueIssuanceAll = await mangata.query.getTotalIssuanceOfTokens();
+
+  expect(valueIssuance).bnEqual(Assets.DEFAULT_AMOUNT.divn(2));
+  expect(valueIssuanceAll[liqId.toString()]).bnEqual(
+    Assets.DEFAULT_AMOUNT.divn(2)
+  );
+});
+
+test("check parameters of getChain function", async () => {
+  const infoChain = await mangata.rpc.getChain();
+
+  expect(infoChain).toContain("Mangata Rococo");
```

instead of hardcode, query it from the node and compare, because this test will fail on kusama setups

---

done